lib/tasks/data.rake

```diff
@@ -0,0 +1,65 @@
+namespace :data do
+  require 'open-uri'
+  require 'json'
```

Comment:
I prefer `require`s in the first lines of the file and I think that's consistent with the rest of the project.